From 976d00f76dd4b5b689ade03244788ea4fd2d7ead Mon Sep 17 00:00:00 2001
From: Denis Barbier <bouzim@gmail.com>
Date: Wed, 23 May 2012 00:24:34 +0200
Subject: [PATCH] Fix build failure on Alpha

This commit fixes issue #243, it has been successfully
tested on Debian (but with OCE 0.9.1).
---
 inc/MFT_FileRecord.hxx |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: oce/inc/MFT_FileRecord.hxx
===================================================================
--- oce.orig/inc/MFT_FileRecord.hxx
+++ oce/inc/MFT_FileRecord.hxx
@@ -64,7 +64,7 @@
         munmap((caddr_t) mmap_addr,(size_t) mmap_size)
 #endif // sgi
 
-#if defined ( __alpha) || defined ( DECOSF1 )
+#if (defined ( __alpha) || defined ( DECOSF1 )) && !defined(linux)
 #include <sys/mman.h>
 #define MMAP(file_addr,mmap_size,fildes) \
         mmap((caddr_t) 0x1000000000,(size_t) mmap_size,PROT_READ, \
